sources/mysql/infoschema_test.go

```diff
@@ -395,7 +405,7 @@ func TestProcessData_MultiCol(t *testing.T) {
 	}
 	internal.AssertSpSchema(conv, t, expectedSchema, stripSchemaComments(conv.SpSchema))
 	columnLevelIssues := map[string][]internal.SchemaIssue{
-		"c49": []internal.SchemaIssue{
+		"c5": []internal.SchemaIssue{
```

Comment:
Why is this change required? `c49` to `c5` ?

Reply:
since few columns are added, there is a change in column from c49 to c5.